### ISSUE TITLE
Migrate from travis-ci.org to travis-ci.com

### ISSUE THREAD

**Issue by michaelpollak** [Wunderbyte-GmbH/moodle-mod_datalynx#148]
Travis will change in 2021, so we need to migrate our testing.

**Comment by michaelpollak** [comment]
I will pack a few things in here,
a) migrating to travis-ci.com
b) fixing automated testing
c) updating automated testing from abandonned moodlerooms/ to moodlehq/
d) updating to version 3.0 https://moodlehq.github.io/moodle-plugin-ci/UPGRADE-3.0.html

**Comment by michaelpollak** [comment]
Well, the problem I found (one of them) is that global $OUTPUT is not allowed in renderers. I changed all the marked references to $this->output but the renderer failes like that. Reverted problematic changes, will take a break now.

**Good news**: Automated testing is working again, update to version 3.0 is completed, migration to travis-ci.com is completed and we changed to the moodlehq testing regiment.
